connector_prestashop/tests/test_import_partner.py

```diff
@@ -88,7 +88,7 @@ def test_import_partner_batch(self):
             self.assertDictEqual(expected_query, self.parse_qs(request.uri))
 
             delay_record_instance = delay_record_mock.return_value
-            self.assertEqual(5, delay_record_instance.import_record.call_count)
+            self.assertEqual(4, delay_record_instance.import_record.call_count)
```

Comment:
```
FAIL: test_import_partner_batch (odoo.addons.connector_prestashop.tests.test_import_partner.TestImportPartner)
Traceback (most recent call last):
`   File "/home/paco/workspace/ptic-saas-server-10/eggs/freezegun-0.3.11-py2.7.egg/freezegun/api.py", line 658, in wrapper
`     result = func(*args, **kwargs)
`   File "connector_prestashop/tests/common.py", line 71, in _decorated
`     result = func(self, *args, **kwargs)
`   File "connector_prestashop/tests/test_import_partner.py", line 91, in test_import_partner_batch
`     self.assertEqual(5, delay_record_instance.import_record.call_count)
` AssertionError: 5 != 4
```
In a fresh PS there are 3 groups + 1 customer instead of 2.